webapp/tests/forms/steps/test_eligibility_steps.py

```diff
@@ -3242,30 +2921,6 @@ def test_if_post_and_session_data_correct_without_registration_method_and_input_
 
         assert step.render_info.next_url == expected_url
 
-    def test_if_post_and_session_data_correct_with_registration_method_software_and_input_data_correct_then_set_next_step_to_success_step(self, app, correct_session_data_with_registration_method_software):
-        with app.test_request_context(method='POST') as req:
-            req.session = SecureCookieSession(
-                {_ELIGIBILITY_DATA_KEY: create_session_form_data(correct_session_data_with_registration_method_software)})
-            step = EligibilityStepChooser('eligibility').get_correct_step(
-                ForeignCountriesDecisionEligibilityInputFormSteuerlotseStep.name, True,
-                ImmutableMultiDict({'foreign_country_eligibility': 'no'}))
-            expected_url = step.url_for_step(EligibilitySuccessDisplaySteuerlotseStep.name)
-            step.handle()
-
-        assert step.render_info.next_url == expected_url
-
-    def test_if_post_and_session_data_correct_with_registration_method_unknown_and_input_data_correct_then_set_next_step_to_maybe_step(self, app, correct_session_data_with_registration_method_unknown):
-        with app.test_request_context(method='POST') as req:
-            req.session = SecureCookieSession(
-                {_ELIGIBILITY_DATA_KEY: create_session_form_data(correct_session_data_with_registration_method_unknown)})
-            step = EligibilityStepChooser('eligibility').get_correct_step(
-                ForeignCountriesDecisionEligibilityInputFormSteuerlotseStep.name, True,
-                ImmutableMultiDict({'foreign_country_eligibility': 'no'}))
-            expected_url = step.url_for_step(EligibilityMaybeDisplaySteuerlotseStep.name)
-            step.handle()
-
-        assert step.render_info.next_url == expected_url
-
```

Comment:
If that is not part of the comment by @frederike-ramin I would suggest that we have tests to ensure that for the different scenarios of elster account answers the correct page (maybe or success) is shown